### PR TITLE
[DOC] Add datasets readme, change lfs tracking

### PR DESCRIPTION
**Pull request by raykallen**
change lfs tracking in dataset directory, add dataset readme

### CODE REVIEW

File: models/datasets/README.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+## Sensitive Information Detection (SID)
+
+This data contains 2000 synthetic pcap payloads generated to mimic sensitive and benign data found in nested jsons from web APIs and environmental variables. Each row is labeled for the presence or absence of 10 different kinds of sensitive information. The data was generated using the python [faker](https://faker.readthedocs.io/en/master/#) package and lists of most [common passwords](https://github.com/danielmiessler/SecLists/tree/master/Passwords/Common-Credentials). If there is any resemblance to real individuals, it is purely coincidental.
+
+### Sample Training Data
+- [sid-sample-training-data.csv](./training-data/sid-sample-training-data.csv)
+
+### Pipeline Validation Data
+- [sid-validation-data.csv](./validation-data/sid-validation-data.csv)
```
Comment (BartleyR) [inline models/datasets/README.md:150]: Let's add a section at the bottom that makes it clear that these are for testing/evaluation purposes only and not meant for production. Maybe something like:

Morpheus contributors will make every effort to keep datasets up-to-date and accurate. However, the data submitted to this repository is provided on an “as is” basis and there is no warranty or guarantee of any kind that the information is accurate, complete, current or suitable for any particular purpose. It is the responsibility of all persons who use Morpheus to independently confirm the accuracy of the data, information, and results obtained via the Morpheus example workflows.

File: models/datasets/README.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+### Pipeline Validation Data
+The same data in both csv and jsonlines
+
+- [abp-validation-data.csv](./validation-data/abp-validation-data.csv)
+- [abp-validation-data.jsonlines](./validation-data/abp-validation-data.jsonlines)
+
+
+## Digital Fingerprinting (DFP) Data
+
+### DFP Azure Logs
```
Comment (BartleyR) [inline models/datasets/README.md:22]: We should avoid empty headers like this. Use this generic DFP Azure Logs section to talk about the logs, then differentiate in the sub-sub headings below between sample training data, pipeline validation, etc.

File: models/datasets/README.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
+# Morpheus Datasets
```
Comment (BartleyR) [inline models/datasets/README.md:1]: In the example workflows themselves (and/or in the model cards), are we also going to link to these datasets?

**Review by BartleyR — CHANGES_REQUESTED** [review]

File: models/.gitattributes [hunk 4]
```diff
@@ -4,4 +4,5 @@ dfp-models/* filter=lfs diff=lfs merge=lfs -text
 abp-models/* filter=lfs diff=lfs merge=lfs -text
 log-parsing-models/* filter=lfs diff=lfs merge=lfs -text
 root-cause-models/* filter=lfs diff=lfs merge=lfs -text
-datasets/** filter=lfs diff=lfs merge=lfs -text
+datasets/training-data/** filter=lfs diff=lfs merge=lfs -text
+datasets/validation-data/** filter=lfs diff=lfs merge=lfs -text
```
Comment (mdemoret-nv) [inline models/.gitattributes:8]: You should keep this all sorted if you can.

Comment (raykallen) [reply models/.gitattributes:3]: sorted alphabetically

File: models/datasets/README.md (hunk 2 above)
Comment (raykallen) [reply models/datasets/README.md:22]: fixed

File: models/datasets/README.md (hunk 1 above)
Comment (raykallen) [reply models/datasets/README.md:149]: added Disclaimer section at the bottom

File: models/datasets/README.md (hunk 3 above)
Comment (raykallen) [reply models/datasets/README.md:1]: made a new issue for this- https://github.com/nv-morpheus/Morpheus/issues/476

**Review by BartleyR — APPROVED** [review]

**Review by tzemicheal — APPROVED** [review]
LGTM

**Comment by raykallen** [comment]
@gpucibot merge

**Pull request merged**